### DISCUSSION TITLE
Stopping hackers

### DISCUSSION THREAD

**Discussion by Dracko132** [community/community#81668] [Other Feature Feedback, Questions, & Ideas]
So I'm wondering do you all support hackers stealing accounts or do you try an help just wondering

**Comment by Akash1134** [comment]
Hi @Dracko132, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by mukhriddin-dev** [comment]
response #4
